lib/plugins/features/movies/index.js

```diff
@@ -16,6 +16,14 @@ exports.register = (server, options, next) => {
         payload: MovieValidator
       }
     }
+  }, {
+    method: 'GET',
+    path: '/movies',
+    config: {
+      handler: (request, reply) => {
+        reply(Controller.findAll(request.query));
+      }
+    }
```

Comment:
I think the only thing we're missing is a validator to make sure our input is squeeky clean.